### PR TITLE
Search trace

### PR DESCRIPTION
**Pull request by Zhifei-Li**
# Description
search func was finished, still working on scatter plot

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Comment by sonamgupta21** [comment]
CSS Feedback - 
1. Reduce the spacing between the left and right containers.
image

2. Theme and Feedback Buttons are not visible 
image

3. Bubble Chart should be responsive.
image

4. Show only the first 7 digits of the trace id here.
image

5. Make the height of the button 40px 
image

**Comment by sonamgupta21** [comment]
1. Dark mode for this doesn't match with the figma. Also provide border-radius at top on the container.
image

2. Make the font size of this to 20px.
image

3. Make this bold.
image

4. The dropdown style for dark mode should be similar to this.It does not look good right now.
image

5. Selected Service/Operation option should be in purple background.
image

6. 0 hours ago should be replaced with "Current time - Start time" 
image

6. Initially it should show Service - "All" , Operations - "All" , Timerange - "last 3-hours"
image

**Comment by Zhifei-Li** [comment]
> CSS Feedback -
> 
> 1. Reduce the spacing between the left and right containers.
> 
> image
> 2. Theme and Feedback Buttons are not visible
> 
> image
> 3. Bubble Chart should be responsive.
> 
> image
> 4. Show only the first 7 digits of the trace id here.
> 
> image
> 5. Make the height of the button 40px
> 
> image

not sure about the second and third issues

**Comment by sonamgupta21** [comment]
1. For traces with errors, the bubbles are too small, making them not visible.
image

2. Regarding scrolling, please make the entire window scrollable, not just the trace results.

File: static/search-traces.html [hunk 1]
```diff
@@ -47,7 +50,7 @@
 
         <div id="app-side-nav" class="active">
         </div>
-
+        <div id="download-trace"></div>
         <div id="dashboard" class="p-20">
```
Comment (sonamgupta21) [inline static/search-traces.html:54]: If you remove the bottom padding, the scrolling issue with the navbar will be resolved.
make it - class="p-20 pb-0".

**Comment by sunitakawane** [comment]
If I select one service, it should show results for that one service only. Also the operations dropdown should also change as per the service. @sonamgupta21 can you verify this

**Comment by sonamgupta21** [comment]
> If I select one service, it should show results for that one service only. Also the operations dropdown should also change as per the service. @sonamgupta21 can you verify this

yes, the search filter is not working correctly.

**Review by sunitakawane — APPROVED** [review]

**Pull request merged**